### PR TITLE
fix TypeHelpComponent

### PR DESCRIPTION
**Pull request by MehdyKarimpour**

### CODE REVIEW

**Comment by MehdyKarimpour** [comment]
Hi Olmo,
TypeHelpComponent => The position of Contextual Menu is wrong. 
Would you please consider to fix?

**Comment by olmobrutall** [comment]
Does this change fixes it?

**Comment by MehdyKarimpour** [comment]
Hi Olmo, 
This change only fixes the exception related to p.renderContextMenu! that could be undefined. Also reloading the helper UI when typeName is changed. Actually, fixing position of contextual menu looks complicated for me.

File: Signum.React.Extensions/Workflow/WorkflowClient.tsx [hunk 1]
```diff
@@ -439,14 +439,14 @@ export function executeWorkflowSave(eoc: Operations.EntityOperationContext<Workf
     API.saveWorkflow(entity, model, replacementModel)
       .then(packWithIssues => {
         eoc.frame.onReload(packWithIssues.entityPack);
-        (eoc.frame.entityComponent as any).setIssues(packWithIssues.issues);
+        wf.setIssues(packWithIssues.issues);
```
[the remark this answers is not in the document]
Comment (MehdyKarimpour) [reply Signum.React.Extensions/Workflow/WorkflowClient.tsx]: Hi, sorry for delay.
It is defined by you, below of saveAndSetErrors() funtion in WorkflowClient.tsx.

let wf = FunctionalAdapter.innerRef(eoc.frame.entityComponent) as WorkflowHandle;

**Pull request merged**